frontend/src/components/accounts/AccountRow.tsx

```diff
@@ -34,6 +34,10 @@ export default class extends React.Component<Props, State> {
   render() {
     const { accountId, createdAt } = this.props;
     const { amount } = this.state;
+    let Id =
+      accountId.length > 25
+        ? accountId.slice(0, 8) + "..." + accountId.slice(accountId.length - 7)
+        : accountId;
```

Comment:
Let's have it as a helper function so we can use `{shortAccountId(accountId)}` in-place.